### DISCUSSION TITLE
Spectrum vs GitHub Discussions

### DISCUSSION THREAD

**Discussion by SleeplessByte** [react-static/react-static#1428] [General]
I find myself neglecting the Spectrum community all the time, mostly because of time constraints caused by COVID-19 and related things. GitHub has granted me and therefor us access to their GitHub Discussions Beta.

By moving _questions_ over to this platform, it's much easier for everyone to discover related PRs, issues or documentation.

Let's try this out? 

---

The spectrum content isn't _gone_. You can still get there if you have the original URL to the posts. Let me know if you need one. I've manually imported the posts from the past 14d.

**Comment by warloque** [comment]
Github discussions is ok, but it looks like the original spectrum community page is now unavailable

**Reply by SleeplessByte** [reply]
Only the index is unavailable, but I can grab any URL if you're looking for something specific.
